### PR TITLE
Travis: bump go to 1.15.x

### PR DESCRIPTION
**Pull request by vrothberg**
Signed-off-by: Valentin Rothberg <rothberg@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Looks like we need the openssl package installed on the linux box?

**Comment by vrothberg** [comment]
@mtrmac FYI ... let's see if can get this green again

**Comment by vrothberg** [comment]
@rhatdan we're green

File: integration/copy_test.go [hunk 1]
```diff
@@ -1114,10 +1114,12 @@ func (s *CopySuite) TestCopyVerifyingMirroredSignatures(c *check.C) {
 	// Create a signature for mirroring-primary:primary-signed without pushing there. This should be easier than using standalone-sign.
 	signingDir := filepath.Join(topDir, "signing-temp")
 	assertSkopeoSucceeds(c, "", "copy", "--src-tls-verify=false", regPrefix+"primary:unsigned", "dir:"+signingDir)
-	c.Logf("%s", combinedOutputOfCommand(c, "ls", "-laR", signingDir))
+	// Unknown error in Travis: https://github.com/containers/skopeo/issues/1093
+	//	c.Logf("%s", combinedOutputOfCommand(c, "ls", "-laR", signingDir))
```
Comment (mtrmac) [inline integration/copy_test.go:1118]: These are “just” diagnostic logs, but I’m not too happy to just ignore a weird failure like this. Let’s see what https://github.com/containers/skopeo/pull/1096 does.

File: .travis.yml [hunk 2]
```diff
@@ -9,7 +9,7 @@ matrix:
   -  os: osx
 
 go:
-  - 1.13.x
+  - 1.15.x
```
Comment (mtrmac) [inline .travis.yml:12]: (Note that this AFAICS only affects the macOS builds; Linux builds are driven by `Dockerfile` == what `golang-bin` package is installed by `fedora:latest`. But that does end up being 1.15 currently.)

**Comment by vrothberg** [comment]
@mtrmac can we merge this PR to unblock the queue? We can continue investigating the permission error.

**Comment by mtrmac** [comment]
Let’s.

In #1096 I ended up with
```
cannot statx '/': Operation not permitted
```
and I have absolutely no idea what’s going on or how to debug that.

I’m worried that we will run into actually needing that kind of data sooner or later, but with no clear path to finding the root cause, it makes no sense to block.

**Pull request merged**